Documentation/ExtensionArchitecture/CreateNewDistribution/Index.rst

```diff
@@ -108,6 +108,18 @@ database dump and the asset files, while ext:bootstrap_package is the real
 site setup. This ends up with only content related stuff being located in
 :file:`fileadmin`, delivered by ext:introduction.
 
+.. _distribution-kickstart-site:
+
+Site configuration
+-------------
```

Comment:
The underline must be exactly as long as the text above, this is a ReST requirement.